core/src/net/sf/openrocket/simulation/listeners/example/RollControlListener.java

```diff
@@ -78,7 +70,7 @@ public void postStep(SimulationStatus status) throws SimulationException {
 		
 		// Find the fin set named CONTROL
 		FinSet finset = null;
-		for (RocketComponent c : status.getConfiguration().getActiveComponents()) {
+		for (RocketComponent c : status.getConfiguration().getAllComponents()) {
```

Comment:
I know `getActiveComponents` is documented in OR as being deprecated, but a recent PR made some improvements on this, so I wouldn't just blindly replace it with `getAllComponents`. I have a feeling this will break with another recent PR that ensures inactive stages are not taken into account in the simulation calculations. So I would revert this back to `getActiveComponents`.

I'll look into whether the deprecated warning should be removed or not, but you don't have to worry about it in this PR.